### PR TITLE
Implement Match betting

### PR DESCRIPTION
**Pull request by Thor77**
[TURNIERE-231](https://jiratinf16b4.it.dh-karlsruhe.de:8443/browse/TURNIERE-231)

### CODE REVIEW

File: app/services/user_service.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  private
+
+  def validate_bet!(match, team)
+    if team.nil?
+      raise UserServiceError, 'Betting on no team in a playoff match is not supported' unless match.group_match?
+    else
+      raise UserServiceError, 'The given team is not involved in the given match' unless match.teams.include? team
+    end
+    raise UserServiceError, 'This user already created a bet on this match' if match.bets.map(&:user).include? @user
+    raise UserServiceError, "Betting is not supported while match is #{match.state}" \
```
Comment (Malaber) [inline app/services/user_service.rb]: "Betting is not **allowed** while match is #{match.state}" would be better I think

**Review by Malaber — CHANGES_REQUESTED** [review]
I don't see a implementation for returning the bet counts for a match or something like that, is that planned for a later pr or am I blind?

**Comment by Thor77** [comment]
Bet counts for a match are implemented [here](https://github.com/turniere/turniere-backend/pull/58/files#diff-f68d3f57253927025ffac9998b1766e1R8)

**Review by Malaber — DISMISSED** [review]

**Review by Malaber — DISMISSED** [review]
lgtm

**Review by Malaber — APPROVED** [review]
lgtm

**Pull request merged**